pkg/api/2018-09-30-preview/api/types.go

```diff
@@ -89,13 +89,15 @@ const (
 
 // NetworkProfile contains configuration for OpenShift networking.
 type NetworkProfile struct {
+	// VnetCIDR is the subnet used by the Openshift cluster in CIDR format
 	VnetCIDR *string `json:"vnetCidr,omitempty"`
 
-	// PeerVnetID is expected to be empty or match
+	// VnetID and PeerVnetID are expected to be empty or match
 	// `^/subscriptions/[^/]+
 	//   /resourceGroups/[^/]+
 	//   /providers/Microsoft.Network
 	//   /virtualNetworks/[^/]+$`
+	VnetID     *string `json:"vnetId,omitempty"`
```

Comment:
```golang
// VnetID (out): the ID of the Vnet created for the OSA cluster
VnetID     *string `json:"vnetId,omitempty"`

// PeerVnetID (in, optional): ID of a Vnet with which the OSA cluster Vnet should be peered.
// If non-empty, this should match
// `^/subscriptions/[^/]+
//   /resourceGroups/[^/]+
//   /providers/Microsoft.Network
//   /virtualNetworks/[^/]+$`
PeerVnetID *string `json:"peerVnetId,omitempty"`
```

Reply:
[also please apply these changes to the other relevant files]

Reply:
changed godoc comments in v20180930preview API, admin API and internal representation.